### ISSUE TITLE
Pet issues

### ISSUE THREAD

**Issue by iSentrie** [FelisDevelopment/fd-pets-support#5]
1. Spawned pet, doesn't attack on command, instead it walks away.
2. Looks like it somehow interfere with players dragging out of vehicle while they are in a driver seat, it allows players to do this stuff.
3. Constant, but random errors:
`InvokeNative: execution failed: Tried to access invalid entity: 154217
[   script:fd-pets] SCRIPT ERROR: Execution of native 00000000faa3d236 in script host failed: Tried to access invalid entity: 154217
[   script:fd-pets] > DeleteEntity (DeleteEntity.lua:4)`

**Comment by iSentrie** [comment]
@niunisas

**Comment by niunisas** [comment]
Awesome! Thanks for the feedback, I'll check this issues probably on weekend.

**Comment by iSentrie** [comment]
Any news?

**Comment by niunisas** [comment]
Hi, sorry, been busy for quite a while with other work. Will take a look at this issues.

**Comment by iSentrie** [comment]
I see 🤔

**Comment by LethalKingz** [comment]
Can confirm the above issues. Experiencing the same.

I think all these issues are related to SetRelationshipToPlayer related natives.

Can somebody please take a look at this soon? Not meaning to be disrespectful, but as it looks this has been going on for a while now, and we do pay for a finished product.

I'd take a look myself and fix it myself, but also this isn't possible due to most of the code being encrypted.

Once again, really not trying to be disrespectful here. But please either give the chance to people to fix it themselves or try to take a look at it soon.

Thanks!

**Comment by niunisas** [comment]
Hi, yes, really sorry. Had a lot of work, issues and etc.
I'll check tomorrow, latest on wednesday and provide a fixed version.

Sorry for such a long wait.

**Comment by LethalKingz** [comment]
Hey no worries my dude!

Just letting you know I'm also experiencing the same.

Bugs are always a thing and the script is otherwise perfect in every way :)

But unfortunately the "Dragging eachother out of the car" issue is a real gamebreaker, that's why I bumped this thread.

Thanks anyway for the fast reply and checking on it soon, much appreciated!

**Comment by niunisas** [comment]
Yeah, have a few huge projects, so my free was very limited for quite a while.

Gonna come back and provide more resources, as well as support for them. 
Can you provide more details about "dragging each other from the car" issue? How to reproduce it, I think I didn't get that correctly.

**Comment by iSentrie** [comment]
Just spawn a pet and everyone will be able to drag you out of a vehicle,

**Comment by niunisas** [comment]
Hi,

Question, drag you out of the vehicle or drag pet out of the vehicle? Cannot really reproduce it.

**Comment by iSentrie** [comment]
1. Spawn a pet
2. Sit in vehicle
3. Call other player and tell him to sit it
4. He will drag you out of a vehicle instead of sit in (because this script somehow allows players to drag you out of a vehicle and prevents them to sit behind you)

**Comment by LethalKingz** [comment]
Any updates?

I'm fairly certain the bug has something to do with these natives, as I have had the bug before when I was experimenting with them.

![image](https://user-images.githubusercontent.com/59566071/171993580-4f04d8e2-c055-4421-b17e-d7fc4ff007b8.png)

**Comment by niunisas** [comment]
Since it looks like it's fixed now, closing this issue. Sorry taking such a long time to fix it.
